unit-test/cs_compute_tests.c

```diff
@@ -841,7 +857,54 @@ void CS_ComputeApp_Test_Nominal(void)
 
     /* Sets AppInfo.CodeSize = 5, sets AppInfo.CodeAddress = 1, AppInfo.AddressesAreValid = true, and returns
      * CFE_SUCCESS */
-    UT_SetHandlerFunction(UT_KEY(CFE_ES_GetAppInfo), CS_COMPUTE_TEST_CFE_ES_GetAppInfoHandler1, NULL);
+    UT_SetHandlerFunction(UT_KEY(CFE_ES_GetModuleInfo), CS_COMPUTE_TEST_CFE_ES_GetModuleInfoHandler1, NULL);
+
+    /* Set to fail condition "NewChecksumValue != ResultsEntry -> ComparisonValue" */
+    UT_SetDeferredRetcode(UT_KEY(CFE_ES_CalculateCRC), 1, 2);
+
+    /* Execute the function being tested */
+    Result = CS_ComputeApp(&ResultsEntry, &ComputedCSValue, &DoneWithEntry);
```

Comment:
UtAssert_UINT32_EQ(CS_ComputeApp..., CFE_SUCCESS); and remove local Result variable.

Reply:
Good point, thanks!